### PR TITLE
test: [M3-7032] - Add Linode Landing Page Integration Tests for VPC

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add Linode Landing Page Integration Tests for VPC.

## Major Changes 🔄
- Add e2e tests to verify VPC feature in Linode Landing Page.

## How to test 🧪
`yarn cy:run -s "cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts"`

### CODE REVIEW

File: packages/manager/cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts [hunk 1]
```diff
@@ -401,6 +408,73 @@ describe('linode landing checks', () => {
...
+    mockGetLinodes(mockLinodes).as('getLinodes');
+    cy.visitWithLogin('/linodes');
+    cy.wait(['@getFeatureFlags', '@getClientStream', '@getLinodes']);
+
+    const mockVPCs = vpcFactory.buildList(5);
+    mockGetVPCs(mockVPCs).as('getVPCs');
+    ui.nav.findItemByTitle('VPC').should('be.visible').click();
+    cy.wait('@getVPCs');
+
+    // Check VPC landing page details
```
Comment (cpathipa) [inline packages/manager/cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts:445]: @cliu-akamaiI I believe the test coverage is not aligned with what the ticket states.

Actual: 

-  We are validating main sidebar element and VPC landing page.

Expected:
- Confirm that the "VPC" column is shown on the Linodes landing page.
- Confirm that clicking on a VPC redirects the user to that VPC's details page 

CC: @jdamore-linode.

**Comment by coliu-akamai** [comment]
@cpathipa @cliu-akamai @jdamore-linode We've actually removed the VPC column from the Linodes landing page due to the increase in 429s from the configs query, so this test may no longer be needed. Sorry for any confusion caused, I should have checked if there was an integration test ticket!

**Comment by abailly-akamai** [comment]
@jdamore-linode Do you know what the status is for this one? one month of inactivity indicates a stale PR probably needing to be closed?

**Comment by coliu-akamai** [comment]
@abailly-akamai @jdamore-linode I think this PR can be closed because (in addition to it being stale) we most likely will not be adding back the VPC column in the linodes landing page table unless we get a lot of feedback during beta/later that it's useful

**Comment by abailly-akamai** [comment]
thanks @coliu-akamai!

**Comment by cliu-akamai** [comment]
Design changed.

**Pull request closed without merging**